.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+# Contributing to Moira
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to Moira and its packages, which are hosted in the 
+[Moira](https://github.com/moira-alert) Organization on GitHub. These are mostly guidelines, not rules. Use your best 
+judgment, and feel free to propose changes to this document in a pull request.
+
+**Table of Content**
+
+* [How Can I Contribute?](#how-can-i-contribute)
+    * [Submitting an Issue](#submitting-an-issue)
+    * [Contributing a Code](#contributing-a-code)
+* [Style Guides](#style-guides)
+    * [Git Style Guide](#git-style-guide)
+    * [Go Code Style Guide](#go-code-style-guide)
+    * [Testing Strategy](#testing-strategy)
+* [Development](#development)
+
+## How Can I Contribute?
+
+### Submitting an Issue
+
+If you find bugs or you want propose an idea that you think will be suitable for moira you can create an issue. The main 
+issue tracker is https://github.com/moira-alert/moira/issues. Issues that belongs for moira backend and web are placed here. 
+Issues that are related to python client and ansible role should be placed in issue tracker for related repo.
+
+We have a templates for two types of issues: Feature Request and a Bug. If you want to submit a bug please be patient 
+and fill all required fields. This will help us to define the problem. 
+
+Try to follow this advices to fill a great issue:
+
+* Check that issue tracker do not contain issues similar to your.
+* Use a clear and descriptive title for the issue to identify the suggestion.
+* Provide a step-by-step description of the suggested enhancement in as many details as possible.
+* Provide specific examples to demonstrate the steps. Include copy/pasteable snippets which you use in those examples, as Markdown code blocks.
+* Describe the current behavior and explain which behavior you expected to see instead and why.
+* Include screenshots and animated GIFs which help you demonstrate the steps to reproduce issue.
+* Explain why this enhancement would be useful to most Moira users.
+* Specify which version of Moira you're using.
```

Comment:
Can we provide a link to a good example of issue descrtiption?

Reply:
I am not sure in this case. The reason is that we have a lot of perfectly documented issues but those issues do not follow templates introduced in this PR. Probably the solution here will be to edit existent issue to follow template. I believe that this two issues will be a good candidates.

* https://github.com/moira-alert/moira/issues/481
* https://github.com/moira-alert/moira/issues/457

Reply:
Maybe we can leave this for now and add a link later, when we have a new issue that follows the template.

Reply:
Let's do this